app/views/tag/_tags.html.erb

```diff
@@ -0,0 +1,18 @@
+<p>
+  <span id="tags">
```

Comment:
@gaeun I think you didn't quite get it. `id` should be unique in a page. So wrap both the partial with a single `<span>` tags